src/Core/Application/Catalog/Products/CreateProductRequestValidator.cs

```diff
@@ -2,23 +2,23 @@ namespace FSH.WebApi.Application.Catalog.Products;
 
 public class CreateProductRequestValidator : CustomValidator<CreateProductRequest>
 {
-    public CreateProductRequestValidator(IReadRepository<Product> productRepo, IReadRepository<Brand> brandRepo, IStringLocalizer<CreateProductRequestValidator> localizer)
+    public CreateProductRequestValidator(IReadRepository<Product> productRepo, IReadRepository<Brand> brandRepo, IStringLocalizer<FileUploadRequestValidator> T)
```

Comment:
shouldn't this be `IStringLocalizer<CreateProductRequestValidator>` ?

Reply:
As the IStringLocalizer is passed into the `FileUploadRequestValidator(T)`, it needs to be of type `IStringLocalizer<FileUploadRequestValidator>`. See error below:

```
[{
	"resource": "/src/Core/Application/Catalog/Products/CreateProductRequestValidator.cs",
	"owner": "csharp",
	"code": "CS1503",
	"severity": 8,
	"message": "Argument \"1\": Konvertierung von \"Microsoft.Extensions.Localization.IStringLocalizer<FSH.WebApi.Application.Catalog.Products.CreateProductRequestValidator>\" in \"Microsoft.Extensions.Localization.IStringLocalizer<FSH.WebApi.Application.Common.FileStorage.FileUploadRequestValidator>\" nicht möglich. [Application]",
	"source": "csharp",
	"startLineNumber": 17,
	"startColumn": 69,
	"endLineNumber": 17,
	"endColumn": 70
}]```